### PR TITLE
Minor Faith Description Edits

### PR DESCRIPTION
**Pull request by lavillastrangiato**
## About The Pull Request

- Removes Psydon's name from Astrata and Noc's descriptions.
- Removes mention that Dendor is Psydon's son???
- Generally cleans up the descriptions, fixing typos and grammar errors.
- Fixes a single typo on Noc's amulet description.
- On some of the descriptions the things the gods hate or appreciate are slightly edited. Please feel free to complain about them even if you think they are small changes. I accept that these do change vibes slightly.

## Why It's Good For The Game

Just makes things a little neater.

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/datums/gods/patrons/divine_pantheon.dm [hunk 1]
```diff
@@ -43,17 +43,17 @@
...
-	worshippers = "Men of the Sea and Sky, Horrid sea-creechers"
-	sins = "Fear, Hubris, Trying to tame water and storms"
+	worshippers = "Sailors of the Sea and Sky, Horrid Sea-Creachers, Fog Islanders"
+	sins = "Fear, Hubris, Trying to Tame the Seas"
 
 /datum/patron/divine/necra
 	name = "Necra"
 	domain = "Force of Death and Decay"
-	desc = "The Veiled Lady, a feared but respected God who leads the dead."
+	desc = "The Veiled Lady, a feared but respected God and keeper of the dead."
```
Comment (BadAtThisGame302) [inline code/datums/gods/patrons/divine_pantheon.dm:54]: Necra is female, could be Goddess?

Comment (lavillastrangiato) [reply code/datums/gods/patrons/divine_pantheon.dm:54]: changed to "force" instead

**Comment by Mogelix** [comment]
Dendor shouldn't be "steward" of the seas. He's not a clerk or accountant, he is the one who hurls a massive wave at a fisher boat to swallow it whole when its haul is desperately needed in a starving village because they took his boons in the sea for granted.

I would personally like to see the forgotten god decapitalized in all references, as a kind of diminutive term used purposefully to diminish his significance... the Saviour, the Great Hero, the Creator, the All-Father, (and Psydon) these should be the appellations of his own faithful for him- and one of those should be what the 'Psydon' patron is called. The weeper should be somewhere in-between. This is just my personal head canon though.

Otherwise, excellent de-slopification and higher-standard writing.

**Comment by lavillastrangiato** [comment]
@Mogelix A steward is not just a clerk, they're keeper or manager of something. They can also keep/manage something in stead of another, which might make acolytes of Abyssor better-suited to be "stewards of the seas".

The other reason I chose that word is for alliteration value. There are, however, other words I could see how they sound; swayer, sovereign, etc.

**Review by Optimism333 — APPROVED** [review]
Looks like decent description updates overall! Don't have any problems giving it approval.

**Pull request merged**